subprojects/packagefiles/speexdsp/meson.build

```diff
@@ -0,0 +1,85 @@
+ project('speexdsp', 'c',
+  version : '1.2.0',
+  default_options: ['warning_level=2',
+                    'c_std=gnu99',
+                    'buildtype=release'],
+  license : 'bsd-like')
+
+lib_current = 1
+lib_revision = 0
+lib_age = 2
+lib_soversion = 1
+lib_version = '@0@.@1@.@2@'.format(lib_soversion, lib_age, lib_revision)
```

Comment:
Isn't this the same as the project version? Can't you do `lib_version = meson.project_version()`?

Reply:
This auto-tools baggage has been tossed.